libGraphite/quickdraw/pixmap.cpp

```diff
@@ -155,6 +157,42 @@ auto graphite::qd::pixmap::set_pm_table(const uint32_t& pm_table) -> void
 
 // MARK: -
 
+auto graphite::qd::pixmap::build_surface(
+    std::shared_ptr<graphite::qd::surface> surface,
+    const std::vector<uint8_t>& pixel_data,
+    const qd::clut& clut,
+    int64_t offset) -> void
+{
```

Comment:
Nice. I knew some of these things must be fundamentally the same and sharing code internally, given that certain resources are just specialised instances. 

Great to actually get this happening here 👍